docs/docs/how-tos/nebari-aws.md

```diff
@@ -0,0 +1,132 @@
+
+- [Sign up for Amazon Web Services](#sign-up-for-google-cloud-platform);
+- [Set up IAM user and access keys](#authentication);
+- [Initialize Nebari with your IAM user credentials](#nebari-initialize);
+- [Deploy Nebari](#deploying-nebari)
+
+For those already familiar to AWS services, feel free to skip this first step and jump straight to the [Nebari authentication](#authentication) section of this guide.
+
+## Sing up for Amazon Web Services
+
+This documentation assumes that the user is already familiar with [AWS Identity and Access Management (IAM)](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) and has a prior knowledge regarding [AWS billing and cost usage](https://aws.amazon.com/eks/pricing/) for Kubernetes related services.
+
+If you are new to AWS, we advise you to first [sign up for a free account](https://aws.amazon.com/free/free-tier/) to get a better understanding of the platform and its features. Please refer to [Amazon VPC (Virtual Private Cloud)](https://aws.amazon.com/vpc/?nc1=h_ls) and [Amazon EKS Prerequisites](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#eks-prereqs) for more information on account types and prerequisites for managing Kubernetes clusters.
+
+:::warning Warning
+The Nebari deployment on AWS will **NOT** fall into `free tier` usage as some of its inner components will lead to [special charges](https://aws.amazon.com/eks/pricing/). Therefore, we recommend that you check [AWS pricing documentation](https://aws.amazon.com/ec2/pricing/) or contact your cloud administrator for more information. If you provision resources outside of the free tier, you may be charged. We're not responsible for any charges that may incur.
+:::
+
+## Authentication
+
+In order to Nebari make requests against the aws API and create it's infrastructure, an authentication method with the appropriate permissions will be required. The easiest way to do this is using a [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) to allow access over your AWS account and EKS cluster management.
+
+As a [best practice](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#lock-away-credentials), do not use the AWS account root user for any task where it's not required. Instead, create a new IAM user for each person that requires administrator access. Then make those users administrators by placing the users into an "Administrators" user group to which you attach the `AdministratorAccess` managed policy.
+
+If its your first time creating a new IAM user, please refer to [Creating your first IAM admin user and user group](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html) for more information. Otherwise, see [Creating an IAM user in your AWS account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) for more information on how to create a IAM user.
```

Comment:
```suggestion
If it's your first time creating a new IAM user, please refer to [Creating your first IAM admin user and user group](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html) for more information. Otherwise, see [Creating an IAM user in your AWS account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) for more information on how to create an IAM user.
```